util/mkerr.pl

```diff
@@ -595,177 +543,153 @@
  * https://www.openssl.org/source/license.html
  */
 
-#include <stdio.h>
 #include <openssl/err.h>
 #include $hincf
 
-/* BEGIN ERROR CODES */
 #ifndef OPENSSL_NO_ERR
 
-# define ERR_FUNC(func) ERR_PACK($pack_errcode,func,0)
-# define ERR_REASON(reason) ERR_PACK($pack_errcode,0,reason)
-
-static ERR_STRING_DATA ${lib}_str_functs[] = {
+static ${const}ERR_STRING_DATA ${lib}_str_functs[] = {
 EOF
-	# Add each function code: if a function name is found then use it.
-	foreach $i (@function) {
-		my $fn;
-		$i =~ /^${lib}_F_(\S+)$/;
-		$fn = $1;
-		if(exists $ftrans{$fn}) {
-			$fn = $ftrans{$fn};
-		}
-#		print OUT "{ERR_PACK($pack_errcode,$i,0),\t\"$fn\"},\n";
-		if(length($i) + length($fn) > 57) {
-			print OUT "    {ERR_FUNC($i),\n     \"$fn\"},\n";
-		} else {
-			print OUT "    {ERR_FUNC($i), \"$fn\"},\n";
-		}
-	}
-	print OUT <<"EOF";
+
+    # Add each function code: if a function name is found then use it.
+    foreach my $i ( @function ) {
+        my $fn;
+        $i =~ /^${lib}_F_(\S+)$/;
+        $fn = $1;
+        $fn = $ftrans{$fn} if exists $ftrans{$fn};
+        print OUT "    {ERR_PACK($pack_lib, $i, 0),\n     \"$fn\"},\n";
```

Comment:
Actually, I'd prefer us to change the style so that generated code can ignore linelength issues :)  but that's later on I suppose.  Originally I did no linebreak, but I figured there is no way that would ever get approved.